### PR TITLE
ENG-5293 #2024 [YSQL][docs] Unique constraint in CREATE and ALTER TABLE

### PR DESCRIPTION
**Pull request by d-uspenskiy**
Add UNIQUE constraint onto `latest/api/ysql/commands/ddl_create_table` and `latest/api/ysql/commands/ddl_alter_table` pages.
EBNF description is taken from Postgres's official doc

Test plan:
Manual

- start docs web browser
- check diagrams on `latest/api/ysql/commands/ddl_create_table` and `latest/api/ysql/commands/ddl_alter_table` pages

### CODE REVIEW

File: docs/content/latest/api/ysql/commands/ddl_create_table.md [hunk 1]
```diff
@@ -80,32 +80,39 @@ postgres=# CREATE TABLE sample(k1 int,
 Table with check constraint
 
 ```sql
-postgres=# CREATE TABLE student_grade (student_id int,
```
Comment (jj-kim) [inline docs/content/latest/api/ysql/commands/ddl_create_table.md]: Is there a reason for this format change?  You may be able to justify getting rid of the column of whitespace, but I don't think you should add a space after each comma.

File: docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf [hunk 2]
```diff
@@ -143,8 +146,10 @@ references_clause ::= 'REFERENCES' table_name [ ( column_name { ',' column_name
 
 key_action ::= 'NO' 'ACTION' | 'RESTRICT' | 'CASCADE' | 'SET' 'NULL' | 'SET' 'DEFAULT' ;
 
-storage_parameter ::= name [ '=' value] ;
+storage_parameter ::= param_name [ '=' param_value] ;
```
Comment (jj-kim) [inline docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf]: `param_name` and `param_value` don't have accompanying rules.  Can you create them?

File: docs/content/latest/api/ysql/commands/ddl_create_table.md (hunk 1 above)
Comment (d-uspenskiy) [reply docs/content/latest/api/ysql/commands/ddl_create_table.md]: I've checked several pages with example. On wast majority of them there is no space between table name and bracket in `CREATE TABLE` statement. I've removed that space.

**Comment by jj-kim** [comment]
Moved to pull #2214.

**Pull request closed without merging**